packages/graphql_server/src/service/twitter-scraping-service.ts

```diff
@@ -0,0 +1,165 @@
+import axios, { AxiosRequestConfig } from 'axios'
+import dotenv from 'dotenv'
+
+dotenv.config()
```

Comment:
I know that you said that Philipp said to use it before in your other PR. But don't use dotenv. 

The process.env will be set when the server is started by the script env.sh.
To use the process.env variables when testing see my post on slack regarding that